project/src/main/puzzle/restaurant-view.gd

```diff
@@ -28,10 +28,10 @@ var _customer_offscreen_rect_position: Vector2
 var _chef_onscreen_rect_position: Vector2
 var _chef_offscreen_rect_position: Vector2
 
-## Ordered list of int indexes which will be cycled to with the next_creature_index or scroll_to_new_creature methods.
+## Ordered list of int indexes which will be cycled to with the next_customer_index or scroll_to_new_creature methods.
```

Comment:
Should this be 'scroll_to_new_customer'?

Reply:
Changed to `scroll_to_new_customer`. Also changed many other similar methods from `creature` to `customer`.